extensions/spring/spring-boot-autoconfigure/pom.xml

```diff
@@ -18,7 +18,7 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <parent>
         <groupId>org.axonframework</groupId>
-        <artifactId>axon</artifactId>
+        <artifactId>axon-framework-extension-spring</artifactId>
```

Comment:
Is it correct that a;; modules `spring-boot-autoconfigure` and `spring-boot-starter`, `spring` have same `artifactId`?